plugins/woocommerce-blocks/bin/webpack-entries.js

```diff
@@ -100,6 +100,10 @@ const blocks = {
 		isExperimental: true,
 		customDir: 'product-filters/inner-blocks/stock-filter',
 	},
+	'product-filter-stock-chips': {
+		isExperimental: true,
+		customDir: 'product-filters/inner-blocks/stock-chips',
+	},
```

Comment:
I guess this is left over from experiments/debugging?

Reply:
Yeah, fixed with 89ae1bd8fb7b4d3c3f9ba4ad6348f76f05664ee0